libs/utils/android/screen.py

```diff
@@ -155,4 +157,57 @@ def unlock(target):
        System.menu(target)
        System.home(target)
 
+    @staticmethod
+    def record_start(target, out_file,
+                     timestamp=True, max_time=None):
+        """
+        Start recording the screen of the device
+        """
+        log = logging.getLogger('Screen')
+
+        options = []
+
+        # Populate recording options
+        if timestamp:
+            options.append('--bugreport')
+        if max_time:
+            options.append('--time-limit {}'.format(max_time))
+
+        remote_file = target.path.join(target.working_directory,
+                                       os.path.basename(out_file))
+
+        record_cmd = 'screenrecord {} {}'.format(
+            ' '.join(options), remote_file
+        )
+
+        target.adb_root(enable=True, force=True)
```

Comment:
Do we need that ? What about using `target.background(record_cmd, as_root=True)` on line 189 instead ? If it's functionally equivalent, I think this would be _slighlty_ faster since it won't restart the adb server. Not a big improvement though :)